### PR TITLE
fixing_naming_of_course_attributes

### PR DESCRIPTION
**Pull request by brano98**
This pull request does not solve issue 324. This issue is suspended for now. This request only fix naming of attributes.

### CODE REVIEW

File: Entities/Course.cs [hunk 1]
```diff
@@ -5,8 +5,8 @@ namespace FRITeam.Swapify.Entities
 {
     public class Course : BaseEntity
     {
-        public string CourseCode { get; set; }
-        public string CourseName { get; set; }        
+        public string CourseCode { get; set; }
```
Comment (marektavac) [inline Entities/Course.cs:8]: So what is it cource code? Can you add comment explanation here?

**Review by marektavac — APPROVED** [review]
If I check the code, the db model wasn't changed, so we don't have to generate db from scratch...

**Comment by brano98** [comment]
Yes you are right. But those are only test cases and I didn't search for replacement since it is still string.

**Comment by marektavac** [comment]
So where do we store the subject shortcut now? Directly in the course name?

**Comment by brano98** [comment]
No for now, we don't store course shortcut at all. This decision was made because school API does'nt return shortcut except when requesting by student's number.

**Comment by brano98** [comment]
This was also reason this issue is suspended

**Comment by marektavac** [comment]
Thanks for explanation

**Pull request merged**